src/_zkapauthorizer/tests/test_plugin.py

```diff
@@ -397,7 +397,12 @@ def test_mismatched_ristretto_issuer(self, config_text, announcement):
             b"tub.port",
             config_text.encode("utf-8"),
         )
-        config_text = BytesIO()
+        # On Tahoe-LAFS <1.16, the config is written as bytes.
+        # On Tahoe-LAFS >=1.16, the config is written as unicode.
+        #
+        # After support for Tahoe <1.16 support is dropped we probably want to
+        # switch to an io.StringIO here.
+        config_text = StringIO()
```

Comment:
I guess we are depending on the fact that `StringIO.StringIO` automatically encodes `unicode` to `bytes`. That seems reasonable, but is maybe worth calling out. It is probably also worth calling out that this is a `StringIO.StringIO` (I was initially confused by the suggestion to switch to a `io.StringIO` when it is already a (different, as it turns out) `StringIO` here.